geth/peers/cache.go

```diff
@@ -20,9 +20,10 @@ type Cache struct {
 func makePeerKey(peerID discv5.NodeID, topic discv5.Topic) []byte {
 	topicLen := len([]byte(topic))
 	lth := topicLen + len(peerID)
-	key := make([]byte, lth)
-	copy(key[:], topic[:])
-	copy(key[topicLen:], peerID[:])
+	key := make([]byte, lth+1)
+	key[0] = 0x01
```

Comment:
We should think about a helper which will create a proper key per service. A usage would look like this:
```go
makeDBKey(databaseIdx, makePeerKey(peerID, topic))
```

Reply:
can you make a constant out of it? maybe even group of constants, where every bucket will have to be added